lib/quantum.ex

```diff
@@ -1,160 +1,202 @@
 defmodule Quantum do
+  @moduledoc """
+  Defines a quantum runner.
 
-  @moduledoc "A cron-like job scheduler"
+  When used, the quantum runner expects the `:otp_app` as option.
+  The `:otp_app` should point to an OTP application that has
+  the quantum rinner configuration. For example, the quantum runner:
 
-  alias Quantum.Job
-  alias Quantum.Normalizer
-  alias Quantum.Timer
+      defmodule MyApp.Quantum do
```

Comment:
I think the whole naming should be reconsidered. I'm not sure current names make sense and I don't like the ones I introduced.